modules/aruco/test/test_arucodetection.cpp

```diff
@@ -554,4 +555,92 @@ TEST(CV_ArucoBitCorrection, algorithmic) {
     test.safe_run();
 }
 
+TEST(CV_ArucoTutorial, can_find_singlemarkersoriginal)
+{
+    string img_path = samples::findFile("../opencv_contrib/modules/aruco/tutorials/aruco_detection/images"
+                                        "/singlemarkersoriginal.jpg", true, true);
```

Comment:
> samples::findFile

Test code should not use samples data.

> `../opencv_contrib/modules`

Wrong way.

---

Check "text" module tests.

Reply:
fixed!)
check `opencv_contrib/modules/aruco/CMakeLists.txt`